### PR TITLE
Add tips

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

Adds more tips

## Changelog
:cl:
add: more tips, due to popular request
/:cl:

### CODE REVIEW

File: config/tips/joketips.txt [hunk 1]
```diff
@@ -14,4 +14,6 @@ Tip: the Theseus AI takes song requests.
 The combat conga line is the ultimate power move.
 Requisitions will kill you if you ask for a BC as smartgunner
 Pew pew makes beno big hurt
-Occasionally the tip of the round might lie to you. Don't panic, this is normal.
\ No newline at end of file
+Occasionally the tip of the round might lie to you. Don't panic, this is normal.
+Tip: Dont die
```
Comment (Menshin) [inline config/tips/joketips.txt]:
```suggestion
Tip: Don't die 
```

**Comment by spookydonut** [comment]
> 
> 
> Move tips out of the config into /strings like /tg/ does it, refactor code accordingly since this is not sustainable.

Make a forum thread for people to submit tips too, add them to the config directly instead of via pr

**Comment by DominikPanic** [comment]
> 
> 
> > Move tips out of the config into /strings like /tg/ does it, refactor code accordingly since this is not sustainable.
> 
> Make a forum thread for people to submit tips too, add them to the config directly instead of via pr

No. Not in the config. There is zero reason for it to be in the config since it requires manual updates. All strings should be in a separate directory like in the /tg/. This model is not sustainable as it requires a keyolder EACH time it is modified.

**Comment by psykzz** [comment]
image

File: code/controllers/stonedmc/subsystem/ticker.dm [hunk 2]
```diff
@@ -337,4 +345,21 @@ SUBSYSTEM_DEF(ticker)
...
+	var/list/marine_tips = world.file2list("strings/tips/marine.txt")
+	var/list/xeno_tips = world.file2list("strings/tips/xeno.txt")
+	var/list/meta_tips = world.file2list("strings/tips/meta.txt")
+
+	var/list/all_tips = marine_tips + xeno_tips + meta_tips
+
+	var/list/meme_tips = world.file2list("strings/tips/meme.txt")
+	if(all_tips.len && prob(95))
+		tip = pick(all_tips)
+	else if(meme_tips.len)
```
Comment (DominikPanic) [inline code/controllers/stonedmc/subsystem/ticker.dm]:
```suggestion
	else if(length(meme_tips))
```

File: code/controllers/stonedmc/subsystem/ticker.dm [hunk 3]
```diff
@@ -337,4 +345,21 @@ SUBSYSTEM_DEF(ticker)
...
+/datum/controller/subsystem/ticker/proc/send_tip_of_the_round()
+	var/tip
+	var/list/marine_tips = world.file2list("strings/tips/marine.txt")
+	var/list/xeno_tips = world.file2list("strings/tips/xeno.txt")
+	var/list/meta_tips = world.file2list("strings/tips/meta.txt")
+
+	var/list/all_tips = marine_tips + xeno_tips + meta_tips
+
+	var/list/meme_tips = world.file2list("strings/tips/meme.txt")
+	if(all_tips.len && prob(95))
```
Comment (DominikPanic) [inline code/controllers/stonedmc/subsystem/ticker.dm]:
```suggestion
	if(prob(95) && length(all_tips))
```

**Pull request merged**